projects/optic/src/commands/oas/shapes/schema.ts

```diff
@@ -114,12 +114,7 @@ export class Schema {
   }
 
   static isPolymorphic(schema: SchemaObject) {
```

Comment:
uh this was wrong so this block never evaluated true https://github.com/opticdev/optic/blob/main/projects/optic/src/commands/oas/shapes/streams/patches.ts#L26 when an empty object like `schema: {}` was passed in